bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -100,32 +96,31 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                         "is not needed for this pull request.");
                 return;
             }
-            var jbsIssue = issueProject.issue(issue.get().shortId());
-            if (jbsIssue.isEmpty()) {
+            var jbsIssueOpt = issueProject.issue(issue.get().shortId());
+            if (jbsIssueOpt.isEmpty()) {
                 pr.removeLabel(CSR_LABEL);
                 reply.println("determined that a [CSR](https://wiki.openjdk.java.net/display/csr/Main) request " +
```

Comment:
This reply is repeated a lot. Perhaps we should have a method for it like some of the other replies have.

Reply:
Fixed. Moved to method `unneededReply`.